src/components/admin/form-components/withSearchBar.jsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+
+import { cleanupFalcorKeys } from 'lib/falcor/falcor-utilities';
+import { withFalcor } from 'components/hocs/falcor-hocs';
+import { debounce } from 'lib/utilities';
+import { getDisplayName } from 'lib/higher-order-helpers';
+
+export const withSearchableConcept = (
+  fields,
+  formatter,
+  category,
+) => WrappedField => {
+  const debounceTime = 250;
+  class Searchable extends React.Component {
+    constructor(props) {
+      super(props);
+      this.state = {
+        suggestions: [],
+      };
+
+      this.handleChange = this.handleChange.bind(this);
+      this.handleClick = this.handleClick.bind(this);
+
+      this.debounceSuggestions = debounce(query => {
```

Comment:
with the new class properties babel plugin I added we don't need to do this in the constructor anymore, you can move it out and just make it `debounceSuggestions =`